src/test/resources/application-test.yml

```diff
@@ -5,53 +5,18 @@ management.endpoint:
   health.cache.time-to-live: 0
   info.cache.time-to-live: 0
 
-api:
-  base:
-    url:
-      oauth: http://localhost:8090/auth
+api.base.url.oauth: http://localhost:8090/auth
 
 spring:
-  datasource:
-    url: jdbc:hsqldb:mem:nomis-db;sql.syntax_ora=true;get_column_name=false;shutdown=false;sql.nulls_first=false;sql.nulls_order=false
-    username: sa
-    password:
-
-    hikari:
-      maximum-pool-size: 2
   main:
     allow-bean-definition-overriding: true
 
-  jpa:
-    show-sql: true
-    properties.hibernate.format_sql: true
-
-  flyway:
-    enabled: true
-    locations: classpath:/db/migration/nomis/ddl,classpath:/db/migration/data,classpath:/db/migration/nomis/data,classpath:/db/migration/nomis/data-hsqldb
-
 data.compliance:
   deletion.enabled: true
   deceased.deletion.enabled: true
   offender.no.booking.deletion.enabled: true
 
-
-hmpps.sqs:
-  provider: localstack
-  queues:
-    datacompliancerequest:
-      queueName: ${random.uuid}
-      dlqName: ${random.uuid}
-    datacomplianceresponse:
-      queueName: ${random.uuid}
-      dlqName: ${random.uuid}
-  topics:
-    datacomplianceevents:
-      arn: arn:aws:sns:eu-west-2:000000000000:${random.uuid}
-    domainevents:
-      arn: arn:aws:sns:eu-west-2:000000000000:${random.uuid}
-
 logging:
   level:
     ROOT: DEBUG
 
-server.shutdown: immediate
```

Comment:
Wouldn't you still need this for tests? I'm wondering if the tests take ages waiting for a graceful shutdown?

Reply:
Yes that is still needed, I will update. Thanks